capstone-rs/src/test.rs

```diff
@@ -3092,3 +3092,20 @@ fn test_insn_size_and_alignment() {
         assert_eq!(original.id(), transmuted.id());
     }
 }
+
+#[test]
+fn test_insn_from_raw() {
+    use capstone_sys::cs_insn;
+
+    let cs = Capstone::new()
+        .x86()
+        .mode(x86::ArchMode::Mode64)
+        .build()
+        .unwrap();
+
+    let insns = cs.disasm_all(X86_CODE, START_TEST_ADDR).unwrap();
+    for insn in insns.iter() {
+        assert_eq!(format!("{:?}", unsafe { Insn::from_raw(&insn.insn as *const cs_insn) }), format!("{:?}", insn));
```

Comment:
Could you pull the pointer (`cs_insn_ptr`) and instruction you create from the pointer (`insn_from_ptr`) into separate local variables? This line is a little complex to follow.

Reply:
Done.